config/build.ts

```diff
@@ -91,7 +90,7 @@ async function processSnippets(dir, processedSnippets) {
 
     // Helpers:
 
-    async function processAndValidateSnippet(file: SnippetFileInput, dir: string): Promise<SnippetProcessedData> {
+    async function processAndValidateSnippet(file: SnippetFileInput, dir: string, isAllSample: boolean): Promise<SnippetProcessedData> {
```

Comment:
This one makes sense to call isPrivateSample/isPublicSample because it is just handling the samples under "samples" or "private-samples" directory and not processing combined playlist.